### PR TITLE
User order history 2

### PR DESCRIPTION
**Pull request by zoegreene**
1. Minor fix to genres drop-down; no longer needs to include artworks in route
2. Added passwords ("test") to our test user accounts in seed data
3. Added user order history to account page

**Question**: How should I include the artworks purchased in the order? In our associations, I don't see that OrderItem hasMany Artworks. Was there a rationale for this?

### CODE REVIEW

**Review by jamilkabir — APPROVED** [review]

**Review by jamilkabir — COMMENTED** [review]
Looks great

**Review by jamilkabir — COMMENTED** [review]
This looks fine as well.

File: client/components/Account.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
   const { email, isAdmin } = user
   const sessionId = document.cookie.split('=')[1]
   if(!sessionId) return (<Redirect to="/" />)
   return (
-    <div>
-      <p>User Details</p>
+    <div className="account">
+      <h3>Account Details</h3>
       <p>Email: {email} </p>
-      <p>Admin: {isAdmin ? 'Yep' : 'Nope'} </p>
```
Comment (adam-adair) [inline client/components/Account.js:15]: ugh, FINE

File: client/components/UserOrders.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            }) :
+          <p>No past orders. Begin your first order!</p> :
+          <p>No past orders. Begin your first order!</p>
+        }
+      </div>
+    </div>
+  )
+}
+
+export default UserOrders;
```
Comment (adam-adair) [inline client/components/UserOrders.js]: This looks nice! Order Items would make sense here, but I think you said on Slack that you were updating this PR, so I fig that or something like it is coming

File: client/components/ArtFilters.js [hunk 3]
```diff
@@ -19,8 +19,13 @@ const ArtFilters = ({ artworks, artists, genres, changeFilter }) => {
         <option value="DEFAULT">GENRE</option>
         { genres ?
           genres.map(genre => {
+            const numArts = artworks
+              .map(a => a.genres)
+              .map(a => a.map(g => g.id))
+              .filter(a => a.includes(genre.id))
+              .length;
             return (
-              <option value={ genre.id } key={ genre.id }>{ genre.name } ({ genre.artworks.length })</option>
```
Comment (adam-adair) [inline client/components/ArtFilters.js:23]: I think this does some weird stuff w the filters now if you select both an artist and a genre.

File: client/components/Account.js (hunk 1 above)
Comment (zoegreene) [reply client/components/Account.js:15]: sawreh

**Comment by adam-adair** [comment]
looks great!

**Pull request merged**